src/VsixV3/EditorsPackage/Microsoft.VisualStudio.Editors.swr

```diff
@@ -0,0 +1,16 @@
+use vs
+
+package name=Microsoft.VisualStudio.Editors
+        version=15.0.0.0
+        vs.package.chip=neutral
+        vs.package.language=en-us
+        vs.package.type=vsix
```

Comment:
I recall some confusion around whether package type should be vsix / manifest in the case where the swix authoring is wrapping a v2 VSIX. Perhaps @jmarolf or @otawfik-ms can confirm this.